modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/getInstanceSize.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
```

Comment:
`eslint-disable no-console` I think you forgot to delete that line too.

Reply:
done!